tools/upload_to_anaconda_posix.sh

```diff
@@ -7,4 +7,5 @@ echo $(python -c "import sys; print(sys.version)")
 sudo chmod -R a+w /home/travis/.cache
 pip install -q git+https://github.com/Anaconda-Platform/anaconda-client@1.8.0
 upload_to_anaconda
+upload_nightlies_to_anaconda
```

Comment:
This will run both functions. I think it would be better to use the `$NIGHTLY` environment variable like you did above. Does it exist in the environment at this point?

Reply:
ahh, this is used by the travis build (aarch64, s390x, ppc64le) which has not yet been converted to use `NIGHTLY` and `HEAD`, so it is not needed yet.

Reply:
> This will run both functions. I think it would be better to use the `$NIGHTLY` environment variable like you did above. Does it exist in the environment at this point?

Right, for the travis build, I don't think `$NIGHTLY` existed at this point. `upload_to_anaconda`  just finds files to upload by matching the pattern `libs/openblas*.tar.gz`. Running both functions as I implemented them would have lead to the same basic ibehavior, except that nightlies would have had `HEAD` in the right part of their urls instead of the version.



> ahh, this is used by the travis build (aarch64, s390x, ppc64le) which has not yet been converted to use `NIGHTLY` and `HEAD`, so it is not needed yet.

OK, I'll just get rid of these new steps in the travis build.